chore: add BC to change log (#2295)
DevExpress/devextreme-reactive: CHANGELOG.md

* **react-scheduler:** add capability to edit recurring appointments ([#2168](https://github.com/DevExpress/devextreme-reactive/issues/2168)) ([c726b6a](https://github.com/DevExpress/devextreme-reactive/commit/c726b6a)), closes [#2202](https://github.com/DevExpress/devextreme-reactive/issues/2202)

BREAKING CHANGES:

* **react-scheduler:** Now the Editing feature requires one of the two plugins `IntegratedEditing` or `EditingMenu`. These plugins implement editing logic for recurrence appointments. The [EditRecurrenceMenu](https://devexpress.github.io/devextreme-reactive/react/scheduler/docs/reference/edit-recurrence-menu/) provides the capability to choose a method of editing recurrence appointments.

```diff
<Scheduler>
   <EditingState />
+  <EditRecurrenceMenu />
+  {/* or <IntegratedEditing />*/}
   ...
</Scheduler>
```

> The EditingState properties `editingAppointmentId`, `defaultEditingAppointmentId`, `onEditingAppointmentIdChange` are removed use instead `editingAppointment`, `defaultEditingAppointment` and `onEditingAppointmentChange`. These properties are contain [appointmentModel](https://devexpress.github.io/devextreme-reactive/react/scheduler/docs/reference/scheduler/#appointmentmodel) of the editing appointment.

Reviewer: ```suggestion
The following `EditingState` properties were replaced:

- `editingAppointmentId` -> `editingAppointment`
- `defaultEditingAppointmentId` -> `defaultEditingAppointment`
- `onEditingAppointmentIdChange` -> `onEditingAppointmentChange`

The new properties contain an [AppointmentModel](https://devexpress.github.io/devextreme-reactive/react/scheduler/docs/reference/scheduler/#appointmentmodel) of the appointment being edited.
```

* **react-scheduler:** add capability to edit recurring appointments ([#2168](https://github.com/DevExpress/devextreme-reactive/issues/2168)) ([c726b6a](https://github.com/DevExpress/devextreme-reactive/commit/c726b6a)), closes [#2202](https://github.com/DevExpress/devextreme-reactive/issues/2202)

BREAKING CHANGES:

> * **react-scheduler:** Editing logic for recurrent appointments was removed from the `EditingState` plugin. Now, editing requires `EditingState` and either the `IntegratedEditing` or the `EditRecurrenceMenu` plugin. In addition, you can add the [EditRecurrenceMenu](https://devexpress.github.io/devextreme-reactive/react/scheduler/docs/reference/edit-recurrence-menu/) plugin to allow users to select how recurrent appointments are edited.

Reviewer 2: ```suggestion
* **react-scheduler:** Editing logic for recurrent appointments was removed from the `EditingState` plugin. Now, editing requires the `EditingState`, and either the `IntegratedEditing` or the `EditRecurrenceMenu` plugin. In addition, you can add the [EditRecurrenceMenu](https://devexpress.github.io/devextreme-reactive/react/scheduler/docs/reference/edit-recurrence-menu/) plugin to allow users to select how recurrent appointments are edited.
```
